curl/src/main/resources/scala-native/curl.c

```diff
@@ -14,12 +14,16 @@ CURLcode org_http4s_curl_CURLMsg_data_result(CURLMsg *curlMsg) {
   return curlMsg->data.result;
 }
 
-const char * const * org_http4s_curl_get_protocols(){
-  return curl_version_info(CURLVERSION_NOW) -> protocols;
+const char * const * org_http4s_curl_get_protocols(curl_version_info_data *data){
+  return data -> protocols;
 }
 
-unsigned int org_http4s_curl_get_version_num(){
-  return curl_version_info(CURLVERSION_NOW) -> version_num;
+unsigned int org_http4s_curl_get_version_num(curl_version_info_data *data){
+  return data -> version_num;
+}
+
+CURLversion org_http4s_curl_version_now(){
+  return CURLVERSION_FIRST;
```

Comment:
Why did this change from `CURLVERSION_NOW`?

Reply:
`CURLVERSION_NOW` is a typedef to one of the enum cases of type `CURLversion`, which on my system points to `CURLVERSION_ELEVENTH`, however the fields we are using were supported in the first version as well, so I thought using minimum version would be better, as it (possibly) makes compiled binaries more compatible with dynamic libraries; however that might be a premature optimization :sweat_smile:

Reply:
Ah, no that makes good sense to me! Thanks for explaining. Just a bit confusing that the method name doesn't match :)